source/include/microapp/cs_MicroappController.h

```diff
@@ -56,8 +64,12 @@ class MicroappController : public EventListener {
 
 	/**
 	 * Limit maximum number of to be registered service routines for the microapp.
+	 * There can be one registered isr per type of BLE event, of which there are for now 3
+	 * - device scanned
+	 * - device connected
+	 * - device disconnected
 	 */
-	static const uint8_t MICROAPP_MAX_BLE_ISR_COUNT = 8;
+	static const uint8_t MICROAPP_MAX_BLE_ISR_COUNT = 3;
```

Comment:
Shouldn't this be a common define?

Reply:
I think it could be done both ways (either have the microapp be aware of this bluenet limit or have it stay ignorant about it). For now I don't see the microapp doing much with the information, but in the future it could be useful if the microapp code is changed.